docs/development.md

```diff
@@ -6,15 +6,79 @@
 
 ### Cloning the repo
 
-TODO: Describe how to clone, folder structure etc.
+To create a local copy of Presidio repository, follow [Github instructions](https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/cloning-a-repository)
+on how to clone a project using git.
+The project is structured so that:
+ - Each Presidio service has a designated directory. In it, the source code for its
+logic and tests, as well as the source code for serving it as an HTTP service and
```

Comment:
I would consider changing this part a little to:
Each directory contains: 
- source code:
    - Engine - to use the service as python package
    - HTTP service - can be found in app.py/
- integration tests
- unit tests

Reply:
Wanted a more holistic description here, so I don;t want to use 'engine' for example. Also not all have the same structure for tests, will try to make it more clear though